add `synchronizeThemeToRoot` option to `ThemeProvider` (#2687)
iTwin/iTwinUI: apps/website/src/content/docs/future-flags.mdx

<ThemeProvider future={{ themeBridge: true }}>
```

### `synchronizeThemeToRoot`

> Setting this flag to `true` makes the current `ThemeProvider` default [`themeOptions.synchronizeThemeToRoot`](themeprovider#synchronize-theme) to `true`, which makes iTwinUI CSS variables available outside the `ThemeProvider`'s DOM subtree.

Reviewer: Having "…to `true`" twice in the same sentence was tripping me up. What do you think about this?
```suggestion
Enabling this flag sets the current `ThemeProvider` default for [`themeOptions.synchronizeThemeToRoot`](themeprovider#synchronize-theme) to `true`, allowing iTwinUI CSS variables available outside the `ThemeProvider`'s DOM subtree.
```

Author: I was trying to use "default" as a verb, which probably added to the confusion.

[`404de2f`](https://github.com/iTwin/iTwinUI/pull/2687/commits/404de2fa9aa1ba3bd69685ccf7c88582a318a3e9): Updated wording.
